src/components/Slider/Rheostat.tsx

```diff
@@ -351,21 +415,16 @@ class Rheostat extends Component {
     });
   }
 
-  startMouseSlide(ev) {
+  private startMouseSlide(ev: MouseEvent) {
     this.setStartSlide(ev, ev.clientX, ev.clientY);
 
-    if (typeof document.addEventListener === 'function') {
```

Comment:
I had no idea we had code like this 😄

Reply:
this is because it was migrated from Rheostat that supported way older browsers